staging/src/k8s.io/apimachinery/pkg/api/meta/help.go

```diff
@@ -111,13 +111,88 @@ func getItemsPtr(list runtime.Object) (interface{}, error) {
 	}
 }
 
+// ExtractList returns obj's Items element as an array of runtime.Objects.
+// Returns an error if obj is not a List type (does not have an Items member).
+func ExtractList(obj runtime.Object) ([]runtime.Object, error) {
```

Comment:
please keep this in the same location as the original `func ExtractList` to make the diff smaller / easier to review

Reply:
ok~ I have re-adjusted the position, I hope to be able to help you.

Reply:
hmm... it's still pretty tricky to follow. The following order would minimize the diff, I think:

```go
func EachListItem... // keep this in its original location

func EachListItemWithAlloc...

func eachListItem...
```

and

```go
func ExtractList... // keep this in its original location

func ExtractListWithAlloc...

func extractList...
```

example diff at https://github.com/liggitt/kubernetes/commit/95b5cd6e295a2c0e371cd75730c8ff1e6f1c9aa6#diff-003f5ae51d520c7714a1a26fa0a5507c4c7c567454f1f85e64bc07d4521d3249 which makes it easier to see what is being changed

Reply:
ok~ already adjusted. PTAL..thx